frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/capacity-and-nodes.tsx

```diff
@@ -89,10 +91,23 @@ export const EnableTaintNodes: React.FC<EnableTaintNodesProps> = ({ state, dispa
 
   return (
     <Checkbox
-      label={t('ceph-storage-plugin~Enable taint nodes')}
-      description={t(
-        "ceph-storage-plugin~Selected nodes will be dedicated to OpenShift Container Storage's use only",
-      )}
+      label={
+        <>
+          {t('ceph-storage-plugin~Mark nodes as dedicated')}{' '}
+          <Tooltip
+            content={
+              <>
+                {t('ceph-storage-plugin~This will taint the nodes with the')}
+                <code>key: node.ocs.openshift.io/storage</code>, <code>value: true</code>,{' '}
+                {t('ceph-storage-plugin~and')} <code>effect: NoSchedule</code>
+              </>
```

Comment:
Use ,`<Trans/>`
```suggestion
              <Trans ns="">
                {This will taint the nodes with the'
                <code>key: node.ocs.openshift.io/storage</code>, <code>value: true</code>,{' '}
                and <code>effect: NoSchedule</code>
              </Trans>
```